<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStadiumWin</class>
 <widget class="QDialog" name="AddStadiumWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Submit a New Team</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="newNameLE">
     <property name="placeholderText">
      <string>( ͡° ͜ʖ ͡°)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>League Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="aLeagueBtn">
        <property name="text">
         <string>American League</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="nLeagueButn">
        <property name="text">
         <string>National League</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Corresponding Stadium</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="5">
       <widget class="QLineEdit" name="stadAddress">
        <property name="placeholderText">
         <string>¯\_(ツ)_/¯</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLineEdit" name="stadPhone">
        <property name="placeholderText">
         <string>¯\_(ツ)_/¯</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QLineEdit" name="stadNameLE">
        <property name="placeholderText">
         <string>¯\_(ツ)_/¯</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="placeholderText">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="5">
       <widget class="QLineEdit" name="turfType">
        <property name="placeholderText">
         <string>( ͡° ͜ʖ ͡°)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="openDate">
        <property name="placeholderText">
         <string>Day</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>Month</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="5">
       <widget class="QLineEdit" name="stadType">
        <property name="text">
         <string/>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
        <property name="placeholderText">
         <string>¯\_(ツ)_/¯</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>INCEPTION: (ಠ ε ಠ):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Capacity:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="QSpinBox" name="stadCapacity">
        <property name="specialValueText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okBtn">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
